### DISCUSSION TITLE
Equivalent to gh api - making authenticated calls to REST / API endpoints

### DISCUSSION THREAD

**Discussion by jmartasek** [ankitpokhrel/jira-cli#770] [Ideas]
I'd like to have an equivalent functionality to https://cli.github.com/manual/gh_api . In a nutshell - a way to automate Jira even when a particular functionality is not present in `jira-cli`.

This would create an immediate option to fill in functionality that is missing - both the ones that will be worked on eventually or even niche use-cases.

**Comment by micgruber** [comment]
Do you know already when the feature will be released?
